Testing/functional/tests/cypress/integration/ui/pages/vaccineCard.js

```diff
@@ -0,0 +1,545 @@
+const { AuthMethod, localDevUri } = require("../../../support/constants");
```

Comment:
The tests should preferably be split into multiple files, with UI tests (that don't require any external communications) staying here and the other tests moving to `cypress/integration/e2e/vaccineCard/`, perhaps separated into authenticated and public tests.

I think all of the tests where you're not enabling the Immunization module are purely UI.

Reply:
Tests have been split.